### ISSUE TITLE
The `-c` option does not use the non-standard config file

### ISSUE THREAD

**Issue by peter-vandenabeele-axa** [serverless/serverless#7306]
# Bug Report

## Description

1. What did you do?

`foobar.yml` file does NOT exist!
`other-serverless.yml` file exists and is valid

```
sls deploy --config foobar.yml # fails as expected
sls deploy --config other-serverless.yml # uses other-serverless.yml as expected

sls deploy -c foobar.yml # uses the default serverless.yml (while it should fail)
sls deploy -c other-serverless.yml # uses the default serverless.yml (while it should use other-serverless.yml
```

1. What happened?

As expected, an error for the non-existing foobar.yml
```
$ sls deploy --config foobar.yml

  Error --------------------------------------------------

  Error: Config file foobar.yml not found
      at Utils.findServicePath (C:\Users\DPVA519\Workspace\tools\nodejs\node_modules\serverless\lib\classes\Utils.js:121:15)
      at new Serverless (C:\Users\DPVA519\Workspace\tools\nodejs\node_modules\serverless\lib\Serverless.js:37:18)
      at C:\Users\DPVA519\Workspace\tools\nodejs\node_modules\serverless\bin\serverless.js:66:24
      at tryCatcher (C:\Users\DPVA519\Workspace\tools\nodejs\node_modules\serverless\node_modules\bluebird\js\release\util.js:16:23)
      at Promise._settlePromiseFromHandler (C:\Users\DPVA519\Workspace\tools\nodejs\node_modules\serverless\node_modules\bluebird\js\release\promise.js:547:31)
... many more lines of stack
```

But for the `-c` option happily continues, reading from the default `serverless.yml` file
(and correctly installing the whole stack).

```
$ sls deploy -c foobar.yml
Serverless: Generating requirements.txt from Pipfile...
Serverless: Parsed requirements.txt from Pipfile in ....
continues until SUCCESS
```

1. What should've happened?

The `-c foobar.yml` should also have tried to read the non-existant foobar.yml file and failed.

I found this by trying `-c other-serverless.yml` and very confusingly that just used the default serverless.yml file.

The doc and `sls deploy --help` suggest the `--configure` and `-c` are equivalent:

https://serverless.com/framework/docs/providers/aws/cli-reference/deploy#options

1. What's the content of your `serverless.yml` file?

The `foobar.yml` file is non-existant

The serverless.yml and other-serverless.yml are valid config files and just work :-)

1. What's the output you get when you use the `SLS_DEBUG=*` environment variable (e.g. `SLS_DEBUG=* serverless deploy`)

I used the `-c foobar.yml` option since that is the broken feature:

```
$ SLS_DEBUG=* serverless deploy -c foobar.yml
Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command requirements
Serverless: Load command requirements:clean
Serverless: Load command requirements:install
Serverless: Load command requirements:cleanCache
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Invoke deploy
Serverless: Invoke package
Serverless: Invoke aws:common:validate
Serverless: Invoke aws:common:cleanupTempDir
Serverless: Generating requirements.txt from Pipfile...
Serverless: Parsed requirements.txt from Pipfile in ...
```

Similar or dependent issues:

I searched in the Open and Closed issues, but did not really find this issue reported ...

```
  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              12.13.1
     Framework Version:         1.59.3
     Plugin Version:            3.2.5
     SDK Version:               2.2.1
     Components Core Version:   1.1.2
     Components CLI Version:    1.4.0
```

**Comment by medikoo** [comment]
@peter-vandenabeele-axa thanks for reporting. This was fixed with `1.60.5`, see https://github.com/serverless/serverless/pull/7131. Ensure to use latest version of `serverless`.

Also name of the param is `--config` (not `--configure`)

**Comment by peter-vandenabeele-axa** [comment]
Thank you for the very fast and accurate reply :-)

This _really_ was a case of "did you upgrade to the latest version" ?

So, I can confirm this issue is fixed in the current version (1.63.0) of 2020-02-12

```
$ npm update -g serverless
# => successful upgrade to 1.63.0

$ sls --version
Framework Core: 1.63.0
Plugin: 3.3.0
SDK: 2.3.0
Components Core: 1.1.2
Components CLI: 1.4.0

$ sls deploy  -c foobar.yml  # the GOOD outcome here is an Error

  Error --------------------------------------------------
```

Thanks, I am closing this issue, fully resolved.

**Comment by peter-vandenabeele-axa** [comment]
> Also name of the param is `--config` (not `--configure`)

Indeed, thx :-) For clarity, I fixed that in the original Question above.
